analysers/commons/commons.py

```diff
@@ -29,11 +30,14 @@ def computeMetrics(data):
     dataMin = np.min(data).item()
     dataMax = np.max(data).item()
     dataLength = len(data)
-    median = np.percentile(data, 50).item()
-    q1 = np.percentile(data, 25).item()
-    q2 = median
-    q3 = np.percentile(data, 75).item()
-    p95 = np.percentile(data, 95).item()
+    percentilesIndexes = range(0, 101)
```

Comment:
@VincenzoFerme because numpy records percentiles from 0 to 100, so we miss percentile 100 without doing that range